modules/ROOT/pages/platform/security/secure-connections.adoc

```diff
@@ -214,6 +214,50 @@ To enable private endpoints using Azure Private Link:
 
 Please see the link:https://learn.microsoft.com/en-us/azure/private-link/rbac-permissions#private-endpoint[Azure Documentation] for required roles and permissions.
 
+== Private links
+
+This private link section is cloud-agnostic and therefore applicable to all clouds. 
+A private link provides secure network connectivity between your application and AuraDB without exposing traffic to the public internet.
+
+The term “private link” refers to:
+
+* Private Service Connect = Google Cloud platform
+* PrivateLink = AWS
+* Private Link = Azure
+
+The following steps explain the process of setting up a private link connection to securely connect your application to an AuraDB Virtual Dedicated Cloud environment. 
```

Comment:
I find this construct slightly problematic since the steps below don't really describe the setting up of a private link, they describe what happens under the hood, so to speak.
Maybe we could say something like:
```suggestion
The following steps explain the process of establishing a private link connection to securely connect your application to an AuraDB Virtual Dedicated Cloud environment. 
```